skrub/_reporting/_data/templates/column-summaries.css

```diff
@@ -18,21 +20,13 @@
     margin-top: var(--spacing-l);
 }
 
-.column-summary-group > .card {
-    margin: 0;
+/* A bit of space between the bar and the buttons */
+.column-selection-buttons {
+    padding-left: 10pt;
 }
 
-/* On small screens, the cards are on top of each other and all have the
-   same width. On wide screens, the cards can be next to
-   each other and wrap horizontally. */
-
-@media screen and (min-width: 60rem) {
-    .column-summary-group {
-        display: flex;
-        flex-direction: row;
-        flex-wrap: wrap;
-        gap: var(--spacing-l);
-    }
+.column-summary-group > .card {
+    margin: 0;
```

Comment:
the shadows being abruptly cut off looks strange/unintentional so removing the margin is fine but then I think we should also remove the shadows

Reply:
I like the shadows: 
![image](https://github.com/user-attachments/assets/fec42951-7213-46e4-a9d6-b0669b2924d5)